app/forms/AtedForms.scala

```diff
@@ -41,13 +39,13 @@ object AtedForms {
   val lengthZero = 0
   val nameLength = 35
   val phoneLength = 24
-  val telephoneRegex = """^[A-Z0-9)\/(\-*#]+$""".r
+  val telephoneRegex: Regex = """^[A-Z0-9)\/(\-*#]+$""".r
 
   val AreYouAnAgentFalseConstraint: Constraint[AreYouAnAgent] = Constraint({ model =>
-    model.isAgent.isEmpty match {
-      case (false) if !model.isAgent.get => Valid
-      case (false) if model.isAgent.get => Invalid("ated.claim-relief.error.agent-claiming-true", "isAgent")
-      case (true) => Invalid("ated.claim-relief.error.agent-claiming", "isAgent")
+    model.isAgent.isEmpty -> model.isAgent.get match {
```

Comment:
This will throw an exception if isAgent is empty now. The locations of the .get calls were gated behind isEmpty being false.

Reply:
Rough pseudo-code for the right implementation is probably something like this
```model.isAgent match {
    Some(false) => Valid
    Some(true) => Invalid(claiming-true)
    _ => Invalid(claiming)
}```